app/models/merit/action.rb

```diff
@@ -15,7 +15,9 @@
 module Merit
   class Action
     def self.check_unprocessed
-      where(processed: false).map(&:check_all_rules)
+      where(processed: false).find_each do |merit_action|
```

Comment:
WDYT about:
```suggestion
      where(processed: false).find_each(&:check_all_rules)
```

Reply:
This will also address a typo and fix CI

Reply:
looks good!